ui/lib/pki/addon/components/page/pki-issuer-rotate-root.hbs

```diff
@@ -111,6 +111,7 @@
 {{#if (eq this.displayedForm "use-old-settings")}}
   {{#if @newRootModel.id}}
     <PkiInfoTableRows @model={{@newRootModel}} @displayFields={{this.displayFields}} />
+    <ParsedCertificateInfoRows @model={{@newRootModel.parsedCertificate}} />
```

Comment:
In other other templates I added `parsedCertificates` to the fieldNames array and checked for it within the loop. Here I just added it to the template after the other fields are rendered. Is there a preference between the two approaches?

Reply:
I _think_ my preference is to have it in the template (a little easier to read at first glance). However, it's not a strong preference, I could go either way.

Reply:
I think template looping can be hard to follow sometimes. Especially since sometimes we're looping through the object of expanded attributes, and sometimes the array of field names. 

I like that this makes it easy to track down where the parsed data that's displayed is coming from, and we can consistently display it at the bottom without having to worry about the fieldName order

Reply:
Great, thanks for the feedback! I'll add it consistently to the template only 👍